test/GNNGraphs/transform.jl

```diff
@@ -149,6 +149,19 @@ end
     end 
 end
 
+@testset "remove_nodes" begin if GRAPH_T == :coo
+    s = [1, 1, 2, 3]
+    t = [2, 3, 4, 5]
+    g = GNNGraph(s, t, graph_type = GRAPH_T)    
+    gnew = remove_nodes(g, [1])
+    snew = [1, 2]
+    tnew = [3, 4]
+    stest, ttest = edge_index(gnew)
+    @test gnew.num_edges == 2
+    @test gnew.num_nodes == 4
+    @test snew == stest
+    @test tnew == ttest
```

Comment:
could also add another test with ndata, data and eweights.

Reply:
will do

Reply:
how would we want to test ndata?
for the nodes removed would we handle that similar to s and t, or would we remove them from ndata?
also since its not always ndata.x
what code should I write to handle that?

please let me know, thanks

Reply:
if we remove a node the corresponding feature should be removed by each entry of `ndata`. I'm not sure which is the best way, maybe `deleteat!`.

Reply:
The operation should be non-mutating though, so we can just resort to `getobs`.

Reply:
done using `getobs`